getting-started/006-data-sources.md

```diff
@@ -1,8 +1,41 @@
-Data Sources
-=============
+# Data Sources
 
-Mapzen Search is powered by several major open data sets and owes a tremendous debt of gratitude to the individuals and communities which produced them.
+Mapzen Search is powered by several major open data sets and owes a tremendous debt of gratitude to
+the individuals and communities which produced them.
+
+## OpenStreetMap
+
+[OpenStreetMap](https://www.openstreetmap.org/) is a community-driven, editable map of the world. It
+prioritizes local knowledge and individual contributions over bulk imports, which often means it has
+excellent coverage even in remote areas. OpenStreetMap contains information on landmarks, buildings,
+roads, and much more.
+
+## Quattroshapes
+
+[Quattroshapes](http://quattroshapes.com/) provides geometry data for both administrative areas
+(countries, states, cities, and so on), and neighborhoods. The administrative area boundaries are derived from
+authoritative government data, while the neighborhoods are calculated from Foursquare check-ins, and
+geo-tagging data from Flickr. Because of the size of the datasets from Flickr and Foursquare,
+Quattroshapes provides a unique and global dataset for a difficult problem.
+
+Because of its global coverage and high quality, Mapzen Search uses Quattroshapes to determine the
```

Comment:
Jumps right into `administrative areas`, which is a bit jargon-y.


Try this on for size:

Quattroshapes provides us with global coverage of location data for:
- countries
- regions (states/provinces)
- counties
- localities (cities, towns, hamlets, villages)
- and in many places, neighbourhoods.

Originally assembled by Foursquare, Quattroshapes provides not only the organizational hierarchy for nearly any point or address worldwide (town > local government > province > country), but also the borders for each of these places. Mapzen Search uses data from Quattroshapes to apply a consistent hierarchy to our data from other sources, so you can be sure that points of interest have consistent data about the cities, regions, and countries they're located in.